HPXMLtoOpenStudio/resources/airflow.rb

```diff
@@ -1359,6 +1473,123 @@ def self.apply_infiltration_and_ventilation_fans(model, weather, vent_mech, vent
     infil_program = OpenStudio::Model::EnergyManagementSystemProgram.new(model)
     infil_program.setName(Constants.ObjectNameInfiltration + ' program')
 
+    infil_program = apply_infiltration_to_living(living_ach50, living_const_ach, infil_program, weather, has_flue_chimney)
+
+    infil_program.addLine('Set QWHV_ervhrv = 0.0')
+    if not vent_mech_erv_hrv.empty?
+      # Actuators for ERV/HRV
+      sens_name = "#{Constants.ObjectNameERVHRV} sensible load"
+      erv_sens_load_var = OpenStudio::Model::EnergyManagementSystemGlobalVariable.new(model, sens_name.gsub(' ', '_'))
+      erv_sens_load_actuator = create_sens_lat_load_actuator_and_equipment(model, sens_name, @living_space, 0.0, 0.0)
+      lat_name = "#{Constants.ObjectNameERVHRV} latent load"
+      erv_lat_load_var = OpenStudio::Model::EnergyManagementSystemGlobalVariable.new(model, lat_name.gsub(' ', '_'))
+      erv_lat_load_actuator = create_sens_lat_load_actuator_and_equipment(model, lat_name, @living_space, 1.0, 0.0)
+
+      infil_program.addLine("Set #{erv_sens_load_actuator.name} = 0.0")
+      infil_program.addLine("Set #{erv_lat_load_actuator.name} = 0.0")
+
+      # ERV/HRV EMS load model
+      # E+ ERV model is using standard density for MFR calculation, caused discrepancy with other system types.
+      # E+ ERV model also does not meet setpoint perfectly.
```

Comment:
Add to comment that this may now be addressed? https://github.com/NREL/EnergyPlus/issues/7507